packages/webapp/src/browser/modules/docusign/Docusign.component.tsx

```diff
@@ -0,0 +1,228 @@
+import { Accordion, Heading, Link, Lozenge } from '@equisoft/design-elements-react';
+import React, { FunctionComponent, memo, useCallback, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { v4 as uuid } from 'uuid';
+import styled from 'styled-components';
+
+interface EnvelopeData {
+    id: string;
+    title: string;
+    createdDate: Date;
+    status: 'send' | 'completed' | 'declined';
+    reasonText?: string;
+}
+
+// Types for Lozenge variants and icons based on your design system
+type LozengeVariant = 'success' | 'warning' | 'alert';
+type IconType = 'check' | 'send' | 'alertOctagon';
+
+// Status configuration for different envelope statuses with proper typing
+const STATUS_CONFIG: Record<EnvelopeData['status'], { variant: LozengeVariant; icon: IconType }> = {
+    completed: { variant: 'success', icon: 'check' },
+    send: { variant: 'warning', icon: 'send' },
+    declined: { variant: 'alert', icon: 'alertOctagon' },
+};
+
+// Styled components defined outside the component
+
+// Tu n'auras pas besoin de ce 'Container' il représente celui qui se trouve deja dans l'app
+const Container = styled.div`
+    width: 560px;
+`;
+
+const StyledAccordion = styled(Accordion)`
+    .eds-Accordion-button {
```

Comment:
Quand tu utilises les classes, il faut faire référence aux constantes du code et non les écrire directement.
Il faudrait ajouter l'import `import { accordionClasses } from '@equisoft/design-elements-react';`, une fois que tu auras modifié l'export principal du DS.
```suggestion
    .${accordionClasses.button} {
```